gapic/core/artman_logging_external_types.yaml

```diff
@@ -0,0 +1,38 @@
+common:
+  api_name: logging-external-types
+  api_version: ""
+  package_type: grpc_common
+  packaging: google-cloud
+  organization_name: google
+  proto_deps:
+    - google-common-protos
+    - google-iam-v1
+  import_proto_path:
+    - ${GOOGLEAPIS}
+  src_proto_path:
+    - ${GOOGLEAPIS}/google/appengine/legacy
+    - ${GOOGLEAPIS}/google/appengine/logging/v1
+    - ${GOOGLEAPIS}/google/appengine/v1
```

Comment:
We don't want all of the appengine protos, only the logging ones... I'm not sure whether the current syntax allows specifying a path to the proto file itself rather than to a directory. If not, you might need to modify `artman` to allow this.

Reply:
Done

Reply:
A blacklist will be brittle solution since there's no way to update it automatically if a new proto ever gets pushed. Can we update artman to accept files as arguments to `src_proto_path` instead.

Reply:
Done.